### PR TITLE
Separated columns from the requestTable.jsx

### PR DESCRIPTION
**Pull request by ss-deep**
As per @r1g023 suggestion, I’ve moved the columns object to a separate file, requestTableColumns.jsx, to enhance the readability of requestTable.jsx.

### CODE REVIEW

File: client/src/components/UserProfile/requestTableColumns.jsx
Comment (NishaVijai) [inline client/src/components/UserProfile/requestTableColumns.jsx:1]: This is something new for me to learn from your code :-) Thank you.

**Review by NishaVijai — APPROVED** [review]
Thank you for the wonderful update and something new for me to learn from your PR. Thank you @ss-deep :-)

**Review by r1g023 — APPROVED** [review]
You are amazing @ss-deep , looks great, thank you

**Comment by KseniiaRiabova** [comment]
You are a star @ss-deep

**Review by KseniiaRiabova — APPROVED** [review]

**Review by Natkuma01 — APPROVED** [review]

**Review by Valeriusdev — APPROVED** [review]

**Pull request merged**